tensorflow_quantum/core/ops/circuit_execution_ops_test.py

```diff
@@ -224,6 +224,35 @@ def test_simulate_state_with_symbols(self, op_and_sim, n_qubits,
 
         self.assertAllClose(cirq_states, op_states, atol=1e-5, rtol=1e-5)
 
+    @parameterized.parameters(
+        list(
+            util.kwargs_cartesian_product(
+                **{
+                    'op_and_sim': [(op, sim)
+                                   for (op,
+                                        sim) in zip(STATE_OPS[:-1], SIMS[:-1])],
+                })))
+    def test_simulate_state_large(self, op_and_sim):
+        """Test a reasonably large and complex circuit."""
+        op, sim = op_and_sim
+        symbol_names = []
+        circuit_batch, resolver_batch = \
+            util.random_circuit_resolver_batch(
+                cirq.GridQubit.rect(4, 4), 5)
```

Comment:
I think we also need to test the specific case from the original bug, which used a line of 12 qubits

Reply:
The case of (4,4) actually proved to be worse than the (1,12) case in my testing.